scrapy/core/downloader/webclient.py

```diff
@@ -1,5 +1,6 @@
 from time import time
 from urllib.parse import urlparse, urlunparse, urldefrag
+from re import match
```

Comment:
:lipstick: `re` is usually imported as a whole. But in any case, it should still be the first import in the file, to keep things alphabetical.

Reply:
Right, wasn't aware, thanks!